action.yml

```diff
@@ -11,6 +11,13 @@ inputs:
   input:
     description: 'The input path.'
     default: '.'
+    required: false
 runs:
-  using: 'node12'
-  main: './dist/main.js'
+  using: composite
+  steps:
+    - name: push
+      shell: bash
+      env:
+        BUF_TOKEN: ${{ inputs.buf_token }}
+        BUF_INPUT: ${{ inputs.input }}
```

Comment:
Should we make this a positional parameter to the `push.bash` script? We don't use this environment variable elsewhere (like we do for `BUF_TOKEN`).

Reply:
I usually prefer environment variables over positional arguments for non-interactive scripts. I'm happy to change this though.

Reply:
Got it. This one caught me off guard since I had to ask, "Wait, that's not a real environment variable we support, right?" It's nice to have a clear delineation between the environment variables that will be set for commands specified in the action and those that are simply used to influence the command's parameters.

If it isn't too much trouble, separating it out into a positional parameter would be nice.

Reply:
I switched it to positional